webapp/src/Service/ImportExportService.php

```diff
@@ -1108,8 +1109,9 @@ protected function importAccountsTsv(array $content, ?string &$message = null):
             } else if (!in_array($type, $djRoles)) {
```

Comment:
alignment seems wrong here.

Also not sure if you see the annotations but PSR12 wants `elseif` over `else if`